cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
             self.assertTrue(frame.frame in range(6, 10))
         self.assertEqual(i + 1, 4)
 
+    def test_api_v1_tasks_annotations_dump_and_upload_with_datumaro(self):
+        test_name = self._testMethodName
+        # get formats
+        dump_formats = dm.views.get_export_formats()
+
+        for dump_format in dump_formats:
+            if dump_format.ENABLED:
+                dump_format_name = dump_format.DISPLAY_NAME
+
+                # TODO skip failed formats
+                if dump_format_name in [
+                    "CamVid 1.0", # issue #2840 and changed points values
+                    "CVAT for video 1.1", # issue #2923 and #2924
+                    "MOT 1.1", # issue #2925
+                    "MOTS PNG 1.0", # issue #2925 and changed points values
+                    "Datumaro 1.0", # not uploaded
+                    "Segmentation mask 1.1", # changed points values
+                    "WiderFace 1.0", # issue #XXX
+                ]:
+                    continue
+
+                print("dump_format_name:", dump_format_name)
```

Comment:
Consider using `self.subTest()` instead. You can find examples below in the file.

Reply:
Done. Added it actions.